Fix case of an instance of Credential (#148)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

!---
Figure: Issuance using Authorization Code Flow 

> (1) The Wallet uses the Credential Issuer's metadata as described in (#credential-issuer-metadata) to learn what Credential types and formats the Credential Issuer supports and to determine the issuer URL of the OAuth 2.0 Authorization Server the Credential Issuer relies on. In this example the Credential Issuer provides the OAuth 2.0 Authorization Server. However, this specification allows for configurations where the Credential Issuer API and the Authorization Server are separate services, potentially provided by different entities.

Reviewer: To me 'deployments' fits better than 'configurations', it feels like it's about more than just configuration, it's affects the overall architecture too.

Author: I did it, thank you

> (1) The Wallet uses the Credential Issuer's metadata as described in (#credential-issuer-metadata) to learn what Credential types and formats the Credential Issuer supports and to determine the issuer URL of the OAuth 2.0 Authorization Server the Credential Issuer relies on. In this example the Credential Issuer provides the OAuth 2.0 Authorization Server. However, this specification allows for configurations where the Credential Issuer API and the Authorization Server are separate services, potentially provided by different entities.

Author: ```suggestion
(1) The Wallet uses the Credential Issuer's metadata as described in (#credential-issuer-metadata) to learn what Credential types and formats the Credential Issuer supports and to determine the issuer URL of the OAuth 2.0 Authorization Server the Credential Issuer relies on. In this example the Credential Issuer provides the OAuth 2.0 Authorization Server. However, this specification allows for deployments where the Credential Issuer API and the Authorization Server are separate services, potentially provided by different entities.
```

(1b) The Issuer-initiated flow begins as the Credential Issuer generates a Credential Offer for certain Credential(s) that it communicates to the Wallet, for example, as a QR code or as a URI. The Credential Offer contains the Credential Issuer's URL and the information about the Credential(s) being offered. This step is defined in (#credential_offer).

> (2) The Wallet uses the Credential Issuer's metadata as described in (#credential-issuer-metadata) to learn what Credential types and formats the Credential Issuer supports and to determine the issuer URL of the OAuth 2.0 Authorization Server the Credential Issuer relies on. In this example the Credential Issuer provides the OAuth 2.0 Authorization Server. However, this specification allows for deployments where the Credential Issuer API and the Authorization Server are separate services, potentially provided by different entities.

Reviewer 2: Why did you exclude the Credential Issuer endpoint?

Author: Since this is included in the first phrase `The Wallet uses the Credential Issuer's metadata as described in (#credential-issuer-metadata)` I prefer references instead of repeating stuffs again and again but if in your perception I should add something just bring your example here, I'll integrate accordingly

Reviewer 2: Is the credential endpoint always the same as the Issuer metadata?

> (2) The Wallet uses the Credential Issuer's metadata as described in (#credential-issuer-metadata) to learn what Credential types and formats the Credential Issuer supports and to determine the issuer URL of the OAuth 2.0 Authorization Server the Credential Issuer relies on. In this example the Credential Issuer provides the OAuth 2.0 Authorization Server. However, this specification allows for deployments where the Credential Issuer API and the Authorization Server are separate services, potentially provided by different entities.

Author: ```suggestion
(2) The Wallet uses the Credential Issuer's metadata as described in (#credential-issuer-metadata) to learn the Credential Endpoint, what Credential types and formats the Credential Issuer supports and to determine the issuer URL of the OAuth 2.0 Authorization Server the Credential Issuer relies on. In this example the Credential Issuer provides the OAuth 2.0 Authorization Server. However, this specification allows for deployments where the Credential Issuer API and the Authorization Server are separate services, potentially provided by different entities.
```

Author: @paulbastian ^

Reviewer 3: ```suggestion
(2) The Wallet uses the Credential Issuer identifier to fetch the Credential Issuer metadata, as described in (#credential-issuer-metadata). The Wallet needs this metadata to learn about the capabilities of the Credential Issuer, including their supported Credential formats and types. These capabilities also encompass the endpoints for both the Credential Issuer and the Authorization Server (OAuth2) to initiate the request. This specification facilitates deployments where the endpoints for the Credential Issuer and the Authorization Server are provided by different entities. Please note that, in this example, the Credential Issuer and OAuth 2.0 Authorization Server correspond to the same entity.
```

Reviewer 4: I like the suggestions from @Sakurann and @awoie, but I would like to adapt it a bit, e.g. `Credential Issuer identifier` is capitalized, i.e. `Credential Issuer Identifier`. Imo, it is better to remove the part about starting/initiating the request with the same argument that @peppelinux mentioned [above](https://github.com/openid/OpenID4VCI/pull/148/files#r1428047507): You do not need the endpoints of the Credential Issuer in this flow to "start/initiate the request". The first request in the flow is the Authorization Request (aside from fetching the metadata...).

```suggestion
(2) The Wallet uses the Credential Issuer Identifier to fetch the Credential Issuer metadata as described in (#credential-issuer-metadata). The Wallet needs this metadata to learn about the capabilities of the Credential Issuer, including the supported Credential formats and types. These capabilities also encompass the endpoints of both the Credential Issuer and the OAuth 2.0 Authorization Server. This specification facilitates deployments where the endpoints of the Credential Issuer and the Authorization Server are provided by different entities. Note that in this example the Credential Issuer and OAuth 2.0 Authorization Server correspond to the same entity.
```

> (2) The Wallet uses the Credential Issuer's metadata as described in (#credential-issuer-metadata) to learn what Credential types and formats the Credential Issuer supports and to determine the issuer URL of the OAuth 2.0 Authorization Server the Credential Issuer relies on. In this example the Credential Issuer provides the OAuth 2.0 Authorization Server. However, this specification allows for deployments where the Credential Issuer API and the Authorization Server are separate services, potentially provided by different entities.

Reviewer 5: I don't understand what `In this example the Credential Issuer provides the OAuth 2.0 Authorization Server.` means.
We also do not use `Credential Issuer API` anywhere else in the text.
I would like to see something closer to the original text.

```suggestion
(2) The Wallet uses the Credential Issuer's URL to fetch the Credential Issuer metadata as described in (#credential-issuer-metadata). The Wallet needs the metadata to learn the Credential types and formats that the Credential Issuer supports, and to determine the Authorization Endpoint (OAuth 2.0 Authorization Server) as well as Credential Endpoint. This specification enables deployments where the Credential Endpoint and the Authorization Endpoint are provided by different entities. Note that in this example the Credential Issuer and OAuth 2.0 Authorization Server correspond to the same entity.
```

> (2) The Wallet uses the Credential Issuer's metadata as described in (#credential-issuer-metadata) to learn what Credential types and formats the Credential Issuer supports and to determine the issuer URL of the OAuth 2.0 Authorization Server the Credential Issuer relies on. In this example the Credential Issuer provides the OAuth 2.0 Authorization Server. However, this specification allows for deployments where the Credential Issuer API and the Authorization Server are separate services, potentially provided by different entities.

Reviewer 3: ```suggestion
(2) The Wallet uses the Credential Issuer's metadata as described in (#credential-issuer-metadata) to learn what Credential types and formats the Credential Issuer supports and to determine the issuer URL of the OAuth 2.0 Authorization Server the Credential Issuer relies on. In this example the Credential Issuer provides the OAuth 2.0 Authorization Server. However, this specification allows for deployments where the Credential Issuer and the Authorization Server are separate services, potentially provided by different entities.
```

* `proof`: OPTIONAL. Object containing the proof of possession of the cryptographic key material the issued Credential would be bound to.  The `proof` object is REQUIRED if the `proof_types` parameter is non-empty and present in the `credentials_supported` map of the issuer metadata for the requested Credential. The `proof` object MUST contain a following claim:
    * `proof_type`: REQUIRED. String denoting the key proof type. The value of this claim determines other claims in the key proof object and its respective processing rules. Key proof types defined in this specification can be found in (#proof_types).
* `credential_identifier`: REQUIRED when `credential_identifier` was returned from the Token Response. MUST NOT be used otherwise. String that identifies a Credential that is being requested to be issued. When this parameter is used, the `format` parameter and any other Credential format specific set of parameters such as those defined in (#format_profiles) MUST NOT be present.

> * `credential_response_encryption`: OPTIONAL. Object containing information for encrypting the Credential Response. If this request element is not present, the corresponding Credential response returned is not encrypted.

Reviewer 2: ```suggestion
* `credential_response_encryption`: OPTIONAL. Object containing information for encrypting the Credential Response. If this request element is not present, the corresponding Credential Response returned is not encrypted.
```
